setup.py

```diff
@@ -39,7 +39,7 @@
     "mkdocstrings==0.18.0",
     "jinja2<3.1.0",
     "mkdocs-jupyter==0.22.0",
-    "plotnine==0.8.0",
+    "plotnine>=0.8.0",
     "matplotlib>=3.4.3",
 ]
```

Comment:
bump version of cluster-experiments to 0.16.0, I did a release that moved it to 0.15.0